### ISSUE TITLE
Can not connect to local dovecot mailbox

### ISSUE THREAD

**Issue by gnordli** [nextcloud/mail#2633]
### Expected behavior
I tried connecting to a dovecot mailbox. 

### Actual behavior
It looks like it logs onto the mail server, but it instantly disconnects.  
Feb  9 02:57:34 oi-mail1 dovecot: imap-login: Login: user=<xxx@xxxxx>, method=PLAIN, rip=127.0.0.1, lip=127.0.0.1, mpid=3152, TLS, session=<fRreyBueUNB/AAAB>
Feb  9 02:57:34 oi-mail1 dovecot: imap(xxx@xxxxx)<3152><fRreyBueUNB/AAAB>: Logged out in=291 out=1226 deleted=0 expunged=0 trashed=0 hdr_count=0 hdr_bytes=0 body_count=0 body_bytes=0

I tried using the FQDN instead of the local address.

I am able to connect to the mailbox using thunderbird. 

### Mail app

**Mail app version: 1.1.2

**Mailserver or service:**  Locally installed Dovecot

### Server configuration

**Operating system**: Ubunutu 18.04
**Web server:** Apache
**Database:** Mysql
**PHP version:**  7.2
**Nextcloud Version:**  18.0.0.

#### Client configuration
**Browser:** Firefox 72.02
**Operating system:** (Ubuntu 19.10)

thanks,
Geoff

**Comment by ChristophWurst** [comment]
The Nextcloud mail app offers an extensive logging system to make it easier identifying and tracking down bugs. Please enable [debug mode](https://docs.nextcloud.com/server/13/developer_manual/general/devenv.html#enabling-debug-mode) and set the log [level to debug](https://docs.nextcloud.com/server/13/admin_manual/configuration_server/logging_configuration.html) in your admin settings. Then, try to reproduce your issue and take another look at `data/nextcloud.log`, `data/horde_imap.log` and `data/horde_smtp.log`. Remove any sensitive information before posting the logs.

**Comment by gnordli** [comment]
Hi @ChristophWurst 

Sorry for the delay on getting back to you on this.  For some reason I didn't get alert someone posted a reply.  

I was able to figure it out with the debug information.  

The problem is: 
[CANNOT] Invalid mailbox name: Name must not have '/' characters (0.001 + 0.000 secs).
>> Command 7 took 0.0002 seconds.
>> ERROR: mail server explicitly reporting an error.

The dovecot installation I had configured had a folder called virtual/Flagged

I removed that folder and now it works fine.  

thanks,
Geoff
